content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -11,23 +11,28 @@ description: "Create your first Solana onchain program in Anchor."
 ## Summary
 
 - **Programs** on Solana have **instruction handlers** that execute instruction
-  logic.
+  logic. They are the entry point for any operation in a program.
 - **Rust** is the most common language for building Solana programs. The
   **Anchor** framework takes care of common grunt work - like reading data from
   incoming instructions, and checking the right accounts are provided - so you
   can focus on building your Solana program.
 
 ## Lesson
 
-Solana's ability to run arbitrary executable code is part of what makes it so
-powerful. Solana programs, similar to "smart contracts" in other blockchain
-environments, are quite literally the backbone of the Solana ecosystem. And the
-collection of programs grows daily as developers and creators dream up and
-deploy new programs.
+Solana's capacity to execute arbitrary code is a key part of its power. Solana
+programs, akin to "smart contracts" in other blockchain systems, are the very
+foundation of the Solana ecosystem. And as developers and creators continuously
+conceive and deploy new programs, the collection of Solana programs continues to
+expand daily.
```

Comment:
Honestly I think the old version of this sucks - it's full of vague terms like 'backbone' and 'powerful' and we can probably change it a lot more. 

> Solana programs, called "smart contracts", form much of the utility of Solana. Every popular Solana exchange, borrow-lend app, digital art auction house, perps platform, and prediction market is a program.